ext/pgvector.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, curl, postgresql }:
+
+stdenv.mkDerivation rec {
+  pname = "pgvector";
+  version = "0.5.1";
+
+  buildInputs = [ curl postgresql ];
```

Comment:
Does it really need `curl`?

Reply:
nope, that was an error b/c I copied the template from pg_net. thanks!